modules/@apostrophecms/image-widget/views/widget.html

```diff
@@ -0,0 +1,8 @@
+{% set attachment = apos.image.first(data.widget._image) %}
+{% if attachment %}
+  <img {% if data.manager.options.className %} class="{{ data.manager.options.className }}"{% endif %}
```

Comment:
Widget configuration doesn't feel like the right place to set `className` .. I think I would mirror rich text and just give it a class of `apos-image`